Authenticator/Source/TokenListViewController.swift

```diff
@@ -237,6 +240,16 @@ extension TokenListViewController {
                 break
             }
         }
+
+        // If firstInserRow has a value > -1 then a row was inserted
+        if firstInsertRow > -1 {
+            let indexPath = NSIndexPath(forRow: firstInsertRow, inSection: sectionIndex)
+            // Scrolls to the newly inserted token at the smallest row index in the tableView
+            // using the minimum amount of scrolling necessary (.None)
+            tableView.scrollToRowAtIndexPath(indexPath,
+                                             atScrollPosition: .None,
+                                             animated: !preventTableViewAnimations)
```

Comment:
If `preventTableViewAnimations` is false, the guard statement at the top of this method will prevent this code from ever being reached. This can just be `animated: true`.

Looking at it now, `preventTableViewAnimations` is confusingly named. Its purpose is to prevent an insert and delete from being triggered when the user drags a row (see `tableView(_:moveRowAtIndexPath:toIndexPath:)`), so it might be more correctly named `preventTableViewUpdates`.